mlflow_databricks_artifacts/tests/test_repo.py

```diff
@@ -0,0 +1,531 @@
+# -*- coding: utf-8 -*-
```

Comment:
These tests have been very lightly modified from https://github.com/mlflow/mlflow/blob/master/tests/store/artifact/test_databricks_artifact_repo.py:

1. Tests for paginated artifact listing have been removed (see https://github.com/dbczumar/mlflow-databricks-artifacts/pull/1/files#r486555745)

2. Mocks have been updated to account for differences in package structure